concept_map.puml

```diff
@@ -53,12 +60,20 @@ CI -[hidden]down-> Hosting
 Hosting -[hidden]right-> Authentication
 Authentication -[hidden]right-> Firestore
 
-Firebase -[hidden]right-> Core
+Web --up--> Firestore : reads metrics and \l stores app config
+Web --up--> Authentication : authenticates
+Web --up--> Hosting : deployed on
+Web --up--> Analytics : reports analytics data
+Web -down-> Sentry : reports about errors
+
+DartCloudFunctions -up-> CloudFunctions : deployed on
+DartCloudFunctions -[hidden]-> Core : uses
+DartCloudFunctions -up-> Firestore : creates aggregation \l documents
```

Comment:
Consider the above suggestion with `CloudFunctions`. Also, it seems like the aggregation message reveals some unnecessary details here. So, how about that?
```suggestion
DartCloudFunctions -[hidden]-> Core : uses
```